src/fiatconnect/ReviewScreen.tsx

```diff
@@ -0,0 +1,340 @@
+    case CryptoType.cEUR:
+      tokenDisplay = t('celoEuro')
+      break
+    default:
+      tokenDisplay = t('total')
+  }
+  return (
+    <View style={styles.sectionContainer}>
+      <Text style={styles.sectionHeaderText}>
+        {t('fiatConnectReviewScreen.transactionDetails')}
+      </Text>
+      <LineItemRow
+        style={styles.sectionMainTextContainer}
+        textStyle={styles.sectionMainText}
+        title={
+          flow === CICOFlow.CashIn
+            ? t('fiatConnectReviewScreen.cashIn.transactionDetailsAmount')
+            : t('fiatConnectReviewScreen.cashOut.transactionDetailsAmount')
+        }
+        amount={
+          <CurrencyDisplay
+            amount={{
+              value: cicoQuote.quote.cryptoAmount,
+              currencyCode: cicoQuote.quote.cryptoType,
+              localAmount: {
+                value: cicoQuote.quote.fiatAmount,
+                currencyCode: cicoQuote.quote.fiatType,
+                exchangeRate: 1,
+              },
+            }}
+            testID="txDetails-fiat"
+          />
+        }
+      />
+      <LineItemRow
+        title={
+          <>
+            {`${tokenDisplay} @ `}
+            <CurrencyDisplay
+              amount={{ value: new BigNumber('1'), currencyCode: cicoQuote.quote.cryptoType }}
+              showLocalAmount={true}
+            />
+          </>
+        }
+        amount={
+          <TokenDisplay
+            amount={cicoQuote.quote.cryptoAmount}
+            tokenAddress={tokenAddress}
+            showLocalAmount={false}
+            testID="txDetails-crypto"
+          />
+        }
+        style={styles.sectionSubTextContainer}
+        textStyle={styles.sectionSubText}
+      />
+      {!!cicoQuote.fiatAccount.BankAccount?.fee && (
+        // TODO(any): consider using FeeDrawer if we want to show fee type / frequency
+        <LineItemRow
+          title={t('feeEstimate')}
+          amount={
+            <CurrencyDisplay
+              amount={{
+                value: 0,
+                currencyCode: cicoQuote.quote.fiatType,
+                localAmount: {
+                  value: cicoQuote.fiatAccount.BankAccount?.fee,
+                  currencyCode: cicoQuote.quote.fiatType,
+                  exchangeRate: 1,
+                },
+              }}
+              testID="txDetails-fee"
+            />
+          }
+          style={styles.sectionSubTextContainer}
+          textStyle={styles.sectionSubText}
+        />
+      )}
+    </View>
+  )
+}
+
+function PaymentMethod({
+  provider,
+  fiatAccount,
+  fiatAccountSchema,
+  fiatAccountLogo,
+}: {
+  provider: ProviderInfo
+  fiatAccount: any
+  fiatAccountSchema: FiatAccountSchema
+  fiatAccountLogo?: string
+}) {
+  const { t } = useTranslation()
+
+  // TODO(any): consider merging this with other schema specific stuff in a generic
+  // type and create via a factory
+  let displayText: string
+  switch (fiatAccountSchema) {
+    case FiatAccountSchema.AccountNumber:
```

Comment:
accountName is a required field for any FiatAccountSchema so I would imagine no matter what schema it uses, it should always have accountName available

Reply:
We want the institution name followed by last 4 digits of account number here. I think this could potentially be different for different schemas, so think useful to have this here.